src/test/kotlin/no/nav/syfo/testhelper/ExternalMockEnvironment.kt

```diff
@@ -20,18 +20,18 @@ class ExternalMockEnvironment private constructor() {
 
     var environment = testEnvironment()
     val mockHttpClient = mockHttpClient(environment = environment)
-    private val redisConfig = environment.redisConfig
-    val redisCache = RedisStore(
+    private val redisConfig = environment.valkeyConfig
+    val redisCache = ValkeyStore(
         JedisPool(
             JedisPoolConfig(),
             HostAndPort(redisConfig.host, redisConfig.port),
             DefaultJedisClientConfig.builder()
                 .ssl(redisConfig.ssl)
-                .password(redisConfig.redisPassword)
+                .password(redisConfig.valkeyPassword)
                 .build()
         )
     )
-    val redisServer = testRedis(environment)
+    val redisServer = testValkey(environment)
```

Comment:
`cacheServer` eller bare `valkeyServer`? 😊
```suggestion
    val cacheServer = testValkey(environment)
```

Reply:
Vi bruker faktisk en test-redis her (embedded-redis), så tenker det er greit at det fortsatt er reflektert i navnet..